envi/archs/rx/rxtables.py

```diff
@@ -0,0 +1,809 @@
+from .const import *
```

Comment:
I think this file needs a lot of cleanup

Reply:
please make suggestions.  that file looks clean to me.  comments, maybe.  but cleaning?

Reply:
Not as much as when I first looked at this, but:
- We don't need the backslash to denote a line continuation.
- Can I can a comment or something to tell me what the structure of the tuples are? I can guess, but I'd prefer something some substantial (especially given how long the tuples are). 
- As always, I like fully qualified import paths and specific imports. It makes it a lot easier to track what's already been imported. Imports aren't free and it helps make our code play nicer with some python tooling.